geneLevel.R

```diff
@@ -211,49 +372,47 @@ convert_gene_level_calls_to_matrix_ascna <- function(gene_level_calls){
 
 if(!interactive()){
 
-  parser = ArgumentParser()
-  parser$add_argument('-f', '--filenames', type='character', nargs='+', help='list of filenames to be processed.')
-  parser$add_argument('-o', '--outfile', type='character', help='Output filename.')
-  parser$add_argument('-t', '--targetFile', type='character', default='IMPACT468', help="IMPACT341/410/468, or a Picard interval list file of gene target coordinates [default IMPACT468]")
-  parser$add_argument('-m', '--method', type='character', default='reg', help="If scna, creates a portal-friendly scna output file [default reg]")
-  args=parser$parse_args()
-
-  filenames = args$filenames
-  outfile = args$outfile
-  method = args$method
-
-  if(args$targetFile=="IMPACT341") {
-
-    geneTargets=IMPACT341_targets
-
-  } else if(args$targetFile=="IMPACT410") {
-
-    geneTargets=IMPACT410_targets
+    parser = ArgumentParser()
+    parser$add_argument('-f', '--filenames', type='character', nargs='+', help='list cncf files to be processed, to be concatenated into one R data.table.')
+    parser$add_argument('-o', '--outfile', type='character', help='Output filename.')
+    parser$add_argument('-t', '--targetFile', type='character', default='IMPACT468', help="IMPACT341/410/468, or a Picard interval list file of gene target coordinates [default IMPACT468]")
+    parser$add_argument('-m', '--method', type='character', default='reg', help="If scna, creates a portal-friendly scna output file [default reg]")
```

Comment:
As Mark pointed out, this argument name can be misleading i.e. to choose between `cncf` vs `em` methods. But it is really just a flag to enable/disable the creation of portal-friendly CNA matrix. Please refactor as you see fit. Or I can do it after this PR. Thanks!

Reply:
Which argument?

Reply:
The `--method` argument.

Reply:
I can do this in another PR after this one. Marking as resolved.